src/main.js

```diff
@@ -1,3 +1,21 @@
 const core = require('@actions/core');
+const { action_library } = require('./action_library');
 
-core.info('Hello!');
+let library = new action_library();
+
+let scope = core.getInput('scope');
+let createOutputVariables = core.getInput('create-output-variables') == 'true';
+
+let inputFilePath = core.getInput('input-file');
+let inputYaml = library.getFileYaml(inputFilePath);
+
+let environmentYaml = library.getCurrentEnvironmentVars();
+let environmentDictionary = library.buildEnvironmentDictionary(scope, inputYaml, environmentYaml);
+
+console.log('Scoped Variables:', environmentDictionary);
```

Comment:
Is this a `console.log` because you want it to print an object nicely?

Reply:
Yes.  I wanted this to be easily read in the logs.